serving/purgatorio/select-and-train-machine-learning-models/deep-learning-theory/deep-learning-theory.md

```diff
@@ -0,0 +1,324 @@
+
+The most used kind of Neural Network to deal with images and videos. They are thought to be less computationally expensive than vanilla Neural Networks (sharing weight among the layers) and have other useful features that help in dealing with 2D images or videos.
+
+_First look (in order):_
+- [Here](http://cs231n.github.io/convolutional-networks/)'s your reference, love it!
+- [Here](https://ujjwalkarn.me/2016/08/11/intuitive-explanation-convnets/) is an awesome deep explanation. 
+- [Here](https://medium.com/technologymadeeasy/the-best-explanation-of-convolutional-neural-networks-on-the-internet-fbb8b1ad5df8) another super good one.
+- [Here](https://developers.google.com/machine-learning/practica/image-classification/) is a serious CNN tutorial from Google.
+- [Here](http://setosa.io/ev/image-kernels/) you have an amazing interactive demo.
+
+_Second pass:_
+- [CNN Chapter](https://www.deeplearningbook.org/contents/convnets.html).
+
+_Dive Deeper_: Use the latest "survey paper" to choose which papers are worth exploring for your case and have a nice panoramic of the state of the art:
+[A Survey of the Recent Architectures of Deep Convolutional Neural Networks](https://arxiv.org/abs/1901.06032)
+
+_Tips & Best practices:_
+[1](https://ujjwalkarn.me/2016/08/11/intuitive-explanation-convnets/), [2](https://www.topbots.com/14-design-patterns-improve-convolutional-neural-network-cnn-architecture/), [3](https://arxiv.org/abs/1709.02601), [4](https://de.mathworks.com/matlabcentral/answers/362024-convolutional-neural-networks-what-is-the-best-practice-training-approach-using-graphics-cards), [5](http://www.academia.edu/4057996/Best_Practices_for_Convolutional_Neural_Networks_Applied_to_Visual_Document_Analysis), [6](https://www.microsoft.com/en-us/research/publication/best-practices-for-convolutional-neural-networks-applied-to-visual-document-analysis/), [7](https://missinglink.ai/guides/neural-network-concepts/neural-networks-image-recognition-methods-best-practices-applications/), [8](https://machinelearningmastery.com/best-practices-document-classification-deep-learning/).
+
+## Recurrent Neural Networks
+
+Recurrent Neural Network (RNNs) outperform other kinds when dealing with text and time-series data because it implements the concept of "memory" inside the network.  
+
+_First look (in order):_
+- [Here](http://colah.github.io/posts/2015-08-Understanding-LSTMs/), a gentle but detailed explanation.
+- [Here](https://www.superdatascience.com/blogs/the-ultimate-guide-to-recurrent-neural-networks-rnn) is another interesting explanation.
+- [Here](https://www.youtube.com/watch?v=9zhrxE5PQgY) is a video with a more practical approach.
+- [Here](https://becominghuman.ai/a-noobs-guide-to-implementing-rnn-lstm-using-tensorflow-1907a5bbb1fa), a guide to implementing RNN in TensorFlow.
+- [Here](https://medium.com/@erikhallstrm/hello-world-rnn-83cd7105b767), a 7-page long blog post regarding the TensorFlow implementation.
+
```

Comment:
ok

Reply:
I prefer to put this inside the last guide "now go build". Or in the Tools and Libraries one. This guide should be library agnostic

Reply:
Sure, mentioning does not harm and then lead them to another guide or section or location! So there is a balance, otherwise, it comes across as biased to one type of tech!